applications/new_schema/types.py

```diff
@@ -75,6 +78,10 @@ def filter_berth_switch(self, queryset, name, value):
         lookup = "__".join([name, "isnull"])
         return queryset.filter(**{lookup: not value})
 
+    def filter_application_code(self, queryset, name, value):
+        lookup = "__".join([name, "exact"])
+        return queryset.exclude(**{lookup: ""})
```

Comment:
Would this also work if you pass `applicationCode: false`?

Reply:
Hmm, atm it will probably do the same thing with `true` or `false` :thinking:

I guess this would be better:
```
    def filter_application_code(self, queryset, name, value):
        lookup = "__".join([name, "exact"])
        if value:
            return queryset.exclude(**{lookup: ""})
```

Reply:
```suggestion
    def filter_application_code(self, queryset, name, value):
        lookup = "__".join([name, "exact"])
        return queryset.exclude(**{lookup: ""}) if value else queryset.filter(**{lookup: ""})
```

Reply:
Ah, so do we want to find all empty `application_code`s if `value==False`?

Reply:
Wait no, the other way around 🤔  Basically: 
```
applicationCode: true --> applications WITH any code
applicationCode: false --> applications WITHOUT code
```

Reply:
Yea, that is what i meant :smile: `false` -> empty `application_code`s == appilcations without code.